source/blogs/dbc/dbc.rst

```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
+#. Check the Android property value *persist.vendor.sys.usb.adbover* with the following command. The default value is ``dwc``, represents normal ADB over USB (DWC).
 
-Steps to enable ADB over DbC support in Cedalon KBL-NUC (Product Code: BLKNUC7iDNHE)
-------------------------------------------------------------------------------------
+    .. code-block:: bash
 
-1. Check the below android property value, default this have value dwc, which means normal 
-   ADB over USB(DWC) will work.
+        root@intel:~# getprop persist.vendor.sys.usb.adbover
+        dwc
 
-   .. image:: images/getprop_dwc.png
+#. Reset the property value to ``dbc``, then reboot the target system.
 
-2. If step 1 output is "dwc" then set it to "dbc".
+    .. code-block:: bash
 
-   .. image:: images/setprop_dbc.png
+        root@intel:~# setprop persist.vendor.sys.usb.adbover dbc
 
-3. Now restart the target system.
+Connect the Target to the Host System
+-------------------------------------
 
-Connection between Host and Target Machine
-------------------------------------------
+Plug the debug Target to the Host system using a `USB Type-A to Type-A (3.0) SuperSpeed Debug cable <https://www.datapro.net/products/usb-3-0-super-speed-a-a-debugging-cable.html>`_. 
+USB 2.0 Type-A to Type-A cable doesn't work in this case.
 
-Plug the debug Target to Host using USB Type-A to Type-A (3.0) SuperSpeed Debug cable
-(Recommended: https://www.datapro.net/products/usb-3-0-super-speed-a-a-debugging-cable.html). 
-USB 2.0 Type-A to Type-A cable will not work in this case.
+At this point the target should have enumerated as a Debug Device on the Host. This can be
+confirmed with the following command:
 
-At this point the target should have enumerated as a Debug Device on Host. This can be
-confirmed as follow in Host Machine.
+    .. code-block:: bash
 
-   .. image:: images/lsusb.png
+        root@intel:~# lsusb -t
+        /:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci-hcd/10p, 5000M
+            |__ Port 4: Dev 31, If 0, Class=Diagnostic, Driver=usbfs, 5000M
 
-$cat /sys/kernel/debug/usb/devices
+        root@intel:~# cat /sys/kernel/debug/usb/devices
+        T:  Bus=02 Lev=01 Prnt=01 Port=03 Cnt=01 Dev#= 31 Spd=5000 MxCh= 0
+        D:  Ver= 3.00 Cls=00(>ifc ) Sub=00 Prot=00 MxPS= 9 #Cfgs=  1
+        P:  Vendor=1d6b ProdID=0010 Rev= 0.10
+        S:  Manufacturer=Linux Foundation
+        S:  Product=Linux USB Debug Target
+        S:  SerialNumber=DW1724778700007
+        C:* #Ifs= 1 Cfg#= 1 Atr=c0 MxPwr=  0mA
+        I:* If#= 0 Alt= 0 #EPs= 2 Cls=dc(unk. ) Sub=02 Prot=01 Driver=usbfs
+        E:  Ad=01(I) Atr=02(Bulk) MxPS=1024 Ivl=0ms
+        E:  Ad=81(I) Atr=02(Bulk) MxPS=1024 Ivl=0ms
 
-   .. image:: images/sys_usb_devices.png
-   
-.. Note:: Speed should be 5000 (i.e. Spd=5000) and Driver should be usbfs (i.e. Driver=usbfs).
+.. Note:: **Speed** should be *5000* (i.e. Spd=5000) and **Driver** should be *usbfs* (i.e. Driver=usbfs) in the above command output.
```

Comment:
above / previous: 
**Speed** should be *5000* (i.e. Spd=5000) and **Driver** should be *usbfs* (i.e. Driver=usbfs) in the previous command output.